test/FunctionalTests/Microsoft.OData.Core.Tests/Json/MockJsonWriter.cs

```diff
@@ -154,5 +155,143 @@ public void Flush()
         {
             throw new NotImplementedException();
         }
+
+        public Task StartPaddingFunctionScopeAsync()

```

Comment:
nit: You could keep the class shorter by using the shorter form, e.g., public void StartPaddingFunctionScopeAsync() => new NotImplementedException();